### ISSUE TITLE
Show Edit link on closed topics addon

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6189]
### Why this would be helpful

Sometimes I want to edit posts on closed topics. This is easily doable by copying the post link and appending /edit to it, but it would be convenient to just have the Edit button there also.

### How this addon works

This addon adds the Edit button to its usual spot o closed topics.

**Comment by mybearworld** [comment]
This also works for New Scratchers, but I'm not sure if we'd want to do show it for New Scratchers.

**Comment by DNin01** [comment]
The ability to edit your own posts on closed topics isn't intended, so I don't think we should take advantage of it.

**Comment by BroJac5246** [comment]
This is a cool trick, but it could cause abuse of this bug, so I don't think we should spread this information if we can avoid it. For example, I didn't know you could do this until now.

**Comment by Samq64** [comment]
I agree this shouldn't be an option for new Scratchers, but for everyone else, I think this could be useful. I've used the /edit method on at least one cringey post on a topic before it was closed.

**Comment by WorldLanguages** [comment]
I don't really think adding this makes sense, if you already know about this and really want to edit your post in a closed topic (which isn't something that happens every day to even to active forums users) it takes a few seconds to use the browser's address bar. This is not the case with other "hacks" which take more effort to do manually, such as the additional characters in "What I'm Working On", or remixing own projects.

As it's too easy already, realistically all we'd be doing by adding a dedicated addon is telling our users about its existence. Do we really want to do that? The URL trick isn't covered by the extension policy - why not popularize this trick on Scratch itself?

**Comment by mybearworld** [comment]
> take advantage

> abuse of this bug

How could it cause more abuse than editing posts normally?

> As it's too easy already, realistically all we'd be doing by adding a dedicated addon is telling our users about its existence.

Is that actually a bad thing? I think there are a few people who'd make use of this but that don't know that this exists. As said above, this isn't really anything that can be badly abused.

**Comment by rgantzos** [comment]
> > As it's too easy already, realistically all we'd be doing by adding a dedicated addon is telling our users about its existence.
> 
> Is that actually a bad thing? I think there are a few people who'd make use of this but that don't know that this exists. As said above, this isn't really anything that can be badly abused.

What he’s saying is that it isn’t necessary for Scratch Addons to cover it, all it would be doing is pointing out something the users can do, pretty much making it useless after being used once.

**Comment by Samq64** [comment]
> Remixing your own projects doesn't break the policy, and that is still included.

I think his point is that adding `/edit` to the post is a lot easier than sending a request to Scratch's API through the console.
